org.nifty-site-manager.nsm.appdata.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Any reason not to ship it with the project? Other distributions also rely on appdata, desktop and icon files.

Reply:
Also, you need to install all three in correct locations, that's why test build failed.